README.md

```diff
@@ -102,6 +102,20 @@ $ cargo build
 You can confirm your current version of llvm by running `llvmenv version`
 or `llvm-config`
 
+#### Nix
+
+Ante is available in the unstable branch of the [nixpkgs repository](https://search.nixos.org/packages?channel=unstable&show=ante&type=packages&query=ante).
```

Comment:
I think this should be a subsection of the linux section rather than being after the windows section

Reply:
I suppose that would make more sense